packages/react-dnd-html5-backend/src/NativeDragSources.js

```diff
@@ -13,9 +13,12 @@ function getDataFromDataTransfer(dataTransfer, typesToTry, defaultValue) {
 
 const nativeTypesConfig = {
 	[NativeTypes.FILE]: {
-		exposeProperty: 'files',
+		exposeProperty: 'dataTransfer',
 		matchesTypes: ['Files'],
-		getData: dataTransfer => Array.prototype.slice.call(dataTransfer.files),
+		getData: ({ files, items }) => ({
+			files: Array.prototype.slice.call(files),
```

Comment:
do we want both files AND items? or to just use items and check for files within them

Reply:
we do, cos items isn't always available. the consumer will have to fallback to files in that case, which should always be available.